### DISCUSSION TITLE
Change exported data format

### DISCUSSION THREAD

**Discussion by clarknelson** [solspace/craft-freeform#1237] [Feature Requests]
**Describe the bug or issue you're experiencing**

Hello my client would like to adjust how certain data is exported for their reports. For example, Instead of the date being exported as YYYY-MM-DD HH:MM:SS they would like MM/DD/YYYY HH:MM. Do you think something like that might be possible? Thanks :)

**Comment by kjmartens** [comment]
Hi @clarknelson,

This isn't currently possible with Freeform, but we'll note this for future consideration (some sort of settings to control to date formatting output). 🙂 

Another option would be to create your own export type that handles this:
https://docs.solspace.com/craft/freeform/v5/developer/events/export-types/
